the-collab-lab/tcl-69-smart-shopping-list
# [AM] Create Landing Page

## Description

This change creates a landing page that links to the About page, allows users to sign in, and provides some basic design to introduce users to the purpose of the app. This change also moves the list of the user's lists to the List page, and makes some minor design/visual organization changes to the List page, removes Manage List from the navbar (as this page will be completely removed in a future update). Additionally, this change unlinks the Home page from Layout so that it can be styled differently, though it borrows some basic aspects from Layout.

## Related Issue

Closes #55 
Sub-issue of #14 

## Acceptance Criteria

- [x] should have a link to the About page
- [x] should allow the user to sign in or indicate if the user is already signed in
- [x] should not have any details about user's specific lists
- [x] should have some visual details that begin to establish the look and feel of the app

## Notes

## Updates

### Before

![Screen Shot 2024-03-25 at 17 27 39](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/115735046/db036279-23f3-4563-9cfa-7d3d8d29bccd)
![Screen Shot 2024-03-25 at 17 28 09](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/115735046/b88dd7be-f3a8-40a8-8e3d-d12607a9b0d8)
![Screen Shot 2024-03-25 at 17 28 16](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/115735046/912b5ff6-83d4-46b3-b548-d6d5fb36a514)
![Screen Shot 2024-03-25 at 17 28 34](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/115735046/4d0d6f01-4923-46dd-85c3-78f0f20935ae)
![Screen Shot 2024-03-25 at 17 28 44](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/115735046/b9d222cf-7f9b-46d6-9827-848fa60c19f7)

### After

![Screen Shot 2024-03-25 at 17 29 45](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/115735046/d5971b78-d5e9-4fb2-a393-579b480313d7)
![Screen Shot 2024-03-25 at 17 29 04](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/115735046/32e025d8-0f52-491e-82b7-8de6b6491037)
![Screen Shot 2024-03-25 at 17 29 35](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/115735046/b4334419-96cb-400a-8294-8b21752c8a2c)

## Testing Steps / QA Criteria

At the "/" route, users should see the home page design as pictured in the "after" images above. They should be able to click the "here" link to navigate to the About page to learn more about the app. They should also be able to sign in and sign out from this page. If not signed in, there should be text which says "Click 'Sign In' to register and get started: " with a button to do so. Once signed in, there should be text which says "Welcome back [User Name]!" with an adjacent sign out button, and below it, "View your lists [here]" which should take the user to the List page when the "here" button is clicked.

The list page should have a navbar with Home, List, and About options, "Select a list or create a new one" near the top of the page, with instructive text and a field to create a new list. After selecting a list with items, those items should display below.
